src/components/Header/Header.css

```diff
@@ -0,0 +1,93 @@
+:root {
+  --blue-100: rgb(246, 249, 255);
+  --blue-400: rgb(55, 131, 255);
+  --blue-500: rgb(34, 119, 255);
+  --grey-100: rgb(240, 242, 244);
+  --grey-300: rgb(222, 225, 233);
+  --white: #fff;
+}
+
+header {
+  padding: .3em;
+  background: var(--white);
+  border-bottom: 1.5px solid var(--grey-100);
+  display: flex;
+  justify-content: space-between;
+}
+
+.btn {
+  font-size: .9em;
+  font-weight: bold;
+  margin: .3em .5em;
+  padding: .5em;
+  width: 8em;
+  border: 1px solid var(--blue-500);
+  border-radius: 1.5em;
+}
+
+.btn-primary {
+  color: var(--blue-500);
+  background: var(--white);
+}
+
+.btn-primary:hover {
+  color: var(--blue-500);
+  background: var(--blue-100);
+}
+
+.btn-secondary {
+  color: var(--white);
+  background: var(--blue-500);
+}
+
+.btn-secondary:hover {
+  color: var(--white);
+  background: var(--blue-400);
+  border-color: var(--blue-400);
+}
+
+#search-bar {
```

Comment:
We usually don't use IDs for styling. I found this article which sums up the reasons quite well: [Reasons not to use ids in CSS](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4)
Have a read and see what you think :)

Reply:
Cool, had a read through and makes a lot of sense. I've changed it to use classes now